base/linalg/bidiag.jl

```diff
@@ -12,9 +12,25 @@ type Bidiagonal{T} <: AbstractMatrix{T}
         new(dv, ev, isupper)
     end
 end
+"""
+    Bidiagonal(dv, ev, isupper)
+
+Constructs an upper (`isupper=true`) or lower (`isupper=false`) bidiagonal matrix using the
+given diagonal (`dv`) and off-diagonal (`ev`) vectors.  The result is of type `Bidiagonal`
+and provides efficient specialized linear solvers, but may be converted into a regular
+matrix with [`full`](:func:`full`). `ev`'s length must be one less than the length of `dv`.
+"""
 Bidiagonal{T}(dv::AbstractVector{T}, ev::AbstractVector{T}, isupper::Bool) = Bidiagonal{T}(collect(dv), collect(ev), isupper)
 Bidiagonal(dv::AbstractVector, ev::AbstractVector) = throw(ArgumentError("did you want an upper or lower Bidiagonal? Try again with an additional true (upper) or false (lower) argument."))
 
+"""
+    Bidiagonal(dv, ev, uplo)
+
+Constructs an upper (`uplo=U`) or lower (`uplo=L`) bidiagonal matrix using the
```

Comment:
`'U'` or `'L'`. These should eventually be changed to symbols instead of `Char`s for consistency.